.github/CODEOWNERS

```
@@ -139,14 +139,14 @@
 /pkg/dogstatsd/                         @DataDog/agent-metrics-logs
 /pkg/forwarder/                         @DataDog/agent-metrics-logs
```

```suggestion
/pkg/forwarder/                         @DataDog/agent-shared-components
```
All looks good. There was a conversation that maybe this should become responsibility of @DataDog/agent-shared-components. I can see both sides to the argument, so we can revise this at another date.

---

Indeed - I based this on that conversation and the original notes on the split. I think the forwarder should be indeed part of ASC responsibility since it's more generally about sending data rather than processing metrics.